db/migrate/20180410125416_rename_ansible_job_categories.rb

```diff
@@ -0,0 +1,11 @@
+class RenameAnsibleJobCategories < ActiveRecord::Migration[5.1]
+  def change
+    User.as_anonymous_admin do
+      JobTemplate.search_for('name ~ "Ansible Default"').all.each do |job_template|
```

Comment:
I think it would be better to be specific and update only templates we seed. If users cloned them, it's up to them to update the category. Also I think the same is required for  "job_category: Ansible Services" change.